src/server/game/Movement/PathGenerator.cpp

```diff
@@ -536,14 +536,14 @@ void PathGenerator::BuildPointPath(const float *startPoint, const float *endPoin
         /// @todo check the exact cases
         TC_LOG_DEBUG("maps", "++ PathGenerator::BuildPointPath FAILED! path sized %d returned\n", pointCount);
         BuildShortcut();
-        _type = PATHFIND_NOPATH;
+        _type = PathType(_type | PATHFIND_NOPATH);
```

Comment:
what does this change btw ?

Reply:
Prevents removing PATHFIND_SHORTCUT from the path type. I agree it's a bit weird to combine it with NOPATH. As most of the checks are already done on NOPATH